applications/app/controllers/OptInController.scala

```diff
@@ -29,16 +29,12 @@ class OptInController extends Controller {
 
   def handle(feature: String, choice: String) = Action { implicit request =>
     Cached(60)(WithoutRevalidationResult(feature match {
-      case "https" => https.opt(choice)
       case "hsts" => hsts.opt(choice)
```

Comment:
I don't think so! @JustinPinner, do you know if anyone still uses the HTTPS opt-in/out routes?